packages/react-components/src/components/VideoGallery/OverflowGallery.tsx

```diff
@@ -99,6 +102,9 @@ export const OverflowGallery = (props: {
 
   /* @conditional-compile-remove(pinned-participants) */
   if (isNarrow) {
+    // There are no pages for ScrollableHorizontalGallery so we will approximate the first 3 remote
+    // participant tiles are visible
+    onChildrenPerPageChange?.(3);
```

Comment:
is this with the assumption that the first 3 spots will be the ones that are being swapped around?

Reply:
Yes